crates/rpc-server/src/facade.rs

```diff
@@ -110,10 +110,12 @@ impl NoxCCPApi for BackgroundFacade<CCProver> {
     async fn get_proofs_after(&self, proof_idx: ProofIdx) -> Result<Vec<CCProof>, Self::Error> {
         let guard = match self.prover.try_read() {
             Ok(g) => g,
-            Err(e) => {
-                return Err(e).context(
-                    "failed to get prover lock: probably on_active_commitment in progress",
+            Err(_) => {
+                tracing::warn!(
```

Comment:
let's make it trace, Nox is supposed to poll CCP often, and we'll see a lot of such logs

Reply:
Set to debug.
Such log will be emitted only during dataset generation.